src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -70,8 +65,7 @@ public void assertTrueForAll(@NotNull Predicate<ItemStack> condition)
 	 *
 	 * @param condition The condition to check for.
 	 */
-	public void assertTrueForNonNulls(@NotNull Predicate<ItemStack> condition)
-	{
+	public void assertTrueForNonNulls(@NotNull Predicate<ItemStack> condition) {
```

Comment:
You broke the formatting here.
```suggestion
	public void assertTrueForNonNulls(@NotNull Predicate<ItemStack> condition)
	{
```